tests/Tests.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+require_once '../index.php'; // require the file to be tested
+
+use PHPUnit\Framework\TestCase;
+
+class Tests extends TestCase
```

Comment:
If you're not finished here, just add a comment explaining that this is a work in progress